docs/uganka.md

```diff
@@ -0,0 +1,124 @@
+# Oasisova uganka
+
+Slovenska ekipa razvijalcev pri [Oasis Network][oasis-network] je za mlade
+slovenske upe pripravila programersko uganko. Počiva na *tajni* blokovni
+verigi z imenom [Oasis Sapphire][oasis-explorer]. Tvoja naloga je, da
+napišeš kratko skripto, ki se poveže s pametno pogodbo na našem omrežju,
+tajno pridobi vprašanje in nanj odgovori. Če bo odgovor pravilen, osvojiš
+kovančke ROSE! 🌹
+
+## Blockchain, Ethereum, Solidity, Dapp, kaj za vraga?
+
+Poznaš le C? Python? Javascript? Odlično! Vsega ostalega se boš priučil. 🎓
+
+Porazdeljena omrežja niso enostavna reč. Programiranje porazdeljenih
+aplikacij ima veliko omejitev v primerjavi z namiznimi ali spletnimi,
+zato je potrebno imeti odprto glavo. Poznavanje algoritmov
+in podatkovnih struktur, pisanje programske kode, ki učinkovito reši problem,
+iznajdljivost pri iskanju hroščev, natančno sledenje poteku programa z
+vidika varnosti in kriptografske rešitve so cenjene vrline.
+
+## OK. Kaj moram storiti?
+
+1. Najprej se nam pridruži na kanalu `#slovenia` na
+   [Oasisovem Discord strežniku][oasis-discord]. Do kanala pridobiš dostop
+   tako, da v kanalu `#get-your-role` klikneš na slovensko zastavico 🇸🇮.
+   Če želiš, se lahko na kratko predstaviš. Lahko pa napišeš le `živjo` in
+   bomo zadovoljni. Če naletiš na nepremostljivo oviro pri reševanju
+   uganke, ti priskočimo na pomoč. Na neprimerna vprašanja ti bomo odgovorili
+   s še bolj neprimernim odgovorom. 😉
+
+2. Če si nov v blokovnih verigah in bo to tvoja prva porazdeljena aplikacija,
+   preglej [kratek Ethereumov vodič po pametnih pogodbah][ethereum-tutorial]. 💡
+
+3. Bravo! Spoštovanega mojstra v programiranju pametnih pogodb za Ethereum
+   zdaj naprošamo, da si prebere, kako bi sprogramiral *tajno* pametno pogodbo
```

Comment:
```suggestion
3. Bravo! Sedaj, ko obvladaš programiranje pametnih pogodb za Ethereum, poglej še *tajne* pametne pogodbe
```
Tale poved mi je kar dolga, uporabi, če se ti zdi koristno

Reply:
Sem odvisnike zamenjal s prilastki in je krajše ;)